packages/esm-reports-app/src/components/edit-scheduled-report/edit-scheduled-report-form.component.tsx

```diff
@@ -0,0 +1,153 @@
+  }, [reportRequest]);
+
+  const handleSubmit = useCallback(
+    (event) => {
+      event.preventDefault();
+
+      setIsSubmitting(true);
+
+      const scheduleRequest = {
+        uuid: reportRequestUuid ? reportRequestUuid : null,
+        reportDefinition: {
+          parameterizable: {
+            uuid: reportDefinitionUuid,
+          },
+          parameterMappings: reportParameters,
+        },
+        renderingMode: {
+          argument: renderModeUuid,
+        },
+        schedule,
+      };
+
+      const abortController = new AbortController();
+      runReportObservable(scheduleRequest, abortController)
+        .pipe(first())
+        .subscribe(
+          () => {
+            showToast({
+              critical: true,
+              kind: 'success',
+              title: t('reportScheduled', 'Report scheduled'),
+              description: t('reportScheduledSuccessfullyMsg', 'Report scheduled successfully'),
+            });
+            closePanel();
+            setIsSubmitting(false);
+          },
+          (error) => {
+            console.error(error);
+            showToast({
+              critical: true,
+              kind: 'error',
+              title: t('reportScheduledErrorMsg', 'Failed to schedule a report'),
+              description: t('reportScheduledErrorMsg', 'Failed to schedule a report'),
+            });
+            closePanel();
+            setIsSubmitting(false);
+          },
+        );
+    },
+    [closePanel, renderModeUuid, reportRequestUuid, reportRequestUuid, reportParameters, schedule],
+  );
+
+  const handleOnChange = () => {
+    setIgnoreChanges((prevState) => !prevState);
+  };
+
+  const handleCronEditorChange = (cron: string, isValid: boolean) => {
+    setSchedule(isValid ? cron : null);
+  };
+
+  useEffect(() => {
+    setIsSubmittable(!!schedule && !!renderModeUuid);
+  }, [schedule, renderModeUuid]);
+
+  return (
+    <Form className={styles.desktopEditSchedule} onChange={handleOnChange} onSubmit={handleSubmit}>
+      <Stack gap={8} className={styles.container}>
+        <SimpleCronEditor initialCron={initialCron} onChange={handleCronEditorChange} />
+        {reportDefinition &&
+          reportDefinition.parameters.map((parameter) => (
+            <ReportParameterInput
+              parameter={parameter}
+              value={reportRequest?.parameterMappings[parameter.name]}
+              onChange={(parameterValue) => {
+                setReportParameters((state) => ({
+                  ...state,
+                  [parameter.name]: parameterValue,
+                }));
+              }}
+            />
+          ))}
+        <div className={styles.outputFormatDiv}>
+          <Select
+            className={styles.basicInputElement}
+            labelText={t('outputFormat', 'Output format')}
+            onChange={(e) => setRenderModeUuid(e.target.value)}
+            value={renderModeUuid}
+          >
+            <SelectItem value={null} />
```

Comment:
```suggestion
            <SelectItem text={t('selectAnOption', 'Select an option')} value={''}
```

Is usually the pattern for an [empty default option](https://react.carbondesignsystem.com/?path=/docs/components-select--overview#empty-default-option).

Reply:
fixed